echo/index.js

```diff
@@ -115,6 +119,19 @@ function Client() {
     return;
   }
 
+
+  var getErrorCode = function (err, statusCode) {
+    var errorCode;
+    if (err) {
+      errorCode = errorCodesAndLabels.REQUEST_ERROR;
+    } else if (httpStatusToErrorCode[statusCode]) {
+      errorCode = httpStatusToErrorCode[statusCode];
+    } else {
+      errorCode = errorCodesAndLabels.UNKNOWN_ERROR;
+    }
+    return errorCode;
+  }
```

Comment:
Could we remove `var errorCode` from this function completely?

Reply:
We could